### ISSUE TITLE
Recon-all wrapper

### ISSUE THREAD

**Issue by pkashya** [nipy/nipype#2979]
Hello experts,

I'm running recon-all by wrapping it in a Nipype workflow.
(cmdline:
recon-all -all -i {data/T1_files(iterfield)} -openmp 64 -sd . -s seg )
The workflow executes smoothly without errors. But it doesn't save the final output files. There is no 'fsaverage' directory created either.
'Runtime_info.txt' shows no errors either, it shows that the files are being created successfully too.

I decided to then run a pipeline with recon-all node as an input to the bbregister node. Looks like it fails after the recon-all node. It exits the command after the mri_nu_correct step without further information.

Please help, as all my future work is contingent on recon-all running smoothly.

Could you possibly link me to an example that runs recon-all for multiple subjects?

Thanks,
Pratik Kashyap
